test/onnx/onnx_import.in.cpp

```diff
@@ -706,61 +706,144 @@ NGRAPH_TEST(${BACKEND_NAME}, onnx_model_matmul)
     test_case.run();
 }
 
-NGRAPH_TEST(${BACKEND_NAME}, onnx_model_softmax)
+NGRAPH_TEST(${BACKEND_NAME}, onnx_model_softmax_0D)
 {
     auto function = onnx_import::import_onnx_model(
-        file_util::path_join(SERIALIZED_ZOO, "onnx/softmax.prototxt"));
+        file_util::path_join(SERIALIZED_ZOO, "onnx/softmax_0D.prototxt"));
 
-    Inputs inputs;
-    inputs.emplace_back(
-        test::NDArray<float, 3>(
-            {{{1, 2, 3, 4, 5}, {6, 7, 8, 9, 10}, {11, 12, 13, 14, 15}, {16, 17, 18, 19, 20}},
+    auto test_case = ngraph::test::NgraphTestCase(function, "${BACKEND_NAME}");
+    test_case.add_input<float>({3.141592});
+    test_case.add_expected_output<float>({1.0});
+    test_case.run();
+}
 
-             {{21, 22, 23, 24, 25},
-              {26, 27, 28, 29, 30},
-              {31, 32, 33, 34, 35},
-              {36, 37, 38, 39, 40}},
+NGRAPH_TEST(${BACKEND_NAME}, onnx_model_softmax_1D)
+{
+    auto function = onnx_import::import_onnx_model(
+        file_util::path_join(SERIALIZED_ZOO, "onnx/softmax_1D.prototxt"));
 
-             {{41, 42, 43, 44, 45},
-              {46, 47, 48, 49, 50},
-              {51, 52, 53, 54, 55},
-              {56, 57, 58, 59, 60}}})
-            .get_vector());
+    auto test_case = ngraph::test::NgraphTestCase(function, "${BACKEND_NAME}");
+    test_case.add_input<float>({-1.0, 0.0, 1.0});
+    test_case.add_expected_output<float>({0.09003058, 0.24472848, 0.66524094});
+    test_case.run();
+}
+namespace
+{
+    // common input for all Softmax 3D test cases (Shape = {3,4,5})
+    const std::vector<float> SOFTMAX_INPUT = {
```

Comment:
Just an idea: You may create a `static constexpr std::array<float, <array_size>> sfmax = { ... }` and then later use it like this: `test_case.add_input<float>({sfmax.begin(), sfmax.end()});`

Reply:
While the `std::array` is constexpr, this suggested approach causes 2 copies in runtime array->vector & vector->vector. I'd rather leave it as is.